src/components/form/JsonEditor/JsonEditor.tsx

```diff
@@ -0,0 +1,282 @@
+import { useState, ReactNode, useEffect } from 'react'
+import AceEditor from 'react-ace'
+import styled, { css } from 'styled-components'
+import 'ace-builds/src-noconflict/mode-json'
+import 'ace-builds/src-noconflict/ext-language_tools'
+import ace from 'ace-builds/src-noconflict/ace'
+import 'ace-builds/webpack-resolver'
+import 'ace-builds/src-noconflict/mode-json'
+import 'ace-builds/src-noconflict/theme-github'
+// @ts-ignore
+import jsonWorkerUrl from 'file-loader!ace-builds/src-noconflict/worker-json'
+
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { Typography, Tooltip, Icon } from '~/components/designSystem'
+import { theme } from '~/styles'
+
+ace.config.setModuleUrl('ace/mode/json_worker', jsonWorkerUrl)
+
+enum JSON_EDITOR_ERROR_ENUM {
+  invalid = 'invalid',
+}
+export interface JsonEditorProps {
+  label: string
+  name?: string
+  placeholder?: string
+  initialValue?: string
+  value?: string
+  infoText?: string
+  error?: string
+  helperText?: string | ReactNode
+  customInvalidError?: string
+  onBlur?: (props: unknown) => void
+  onChange?: (value: string) => void
+  onError?: (err: keyof typeof JSON_EDITOR_ERROR_ENUM) => void
+}
+
+export const JsonEditor = ({
```

Comment:
No "codeEditor" in the end ?

Reply:
No because it's just for Json, CodeEditor would be good if we could configure the language which is not the case right now